withdrawn-packages.txt

```diff
@@ -1,4 +1,15 @@
-pgvector-0.6.1-r0.apk
-python3-3.11.1-r3.apk
-python3-dev-3.11.1-r3.apk
-python3-doc-3.11.1-r3.apk
+python3-3.10.7-r0.apk
```

Comment:
is this meant to be a historical/complete list of all things withdrawn? or is it only input to a "run" of a tool that withdraws things.

ie, are you supposed to remove 'pgvector' from this list?

Reply:
I believe that after one runs the withdraw packages workflow, these become historic and just "do nothing each time, cannot remove thing" and hence the workflow has `|| true`. I wish the withdraw packages would autogenerate a commit that drops removed packages from this file.

The full list of all withdrawn packages is in the bucket and published at https://packages.wolfi.dev/os/withdrawn-packages.txt see how it has all old versions.